packages/agreement-consumer/kubernetes/dev/deployment.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: interop-be-agreement-consumer-refactor
+  namespace: dev-refactor
+  labels:
+    app: interop-be-agreement-consumer-refactor
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: interop-be-agreement-consumer-refactor
+  template:
+    metadata:
+      labels:
+        app: interop-be-agreement-consumer-refactor
+    spec:
+      serviceAccountName: interop-be-agreement-consumer-refactor
+      containers:
+        - name: interop-be-agreement-consumer-refactor
+          image: ghcr.io/pagopa/agreement-consumer@$IMAGE_DIGEST
+          imagePullPolicy: Always
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
```

Comment:
Are we running a web server on the consumer?

Reply:
it could be usefull if liveness-check API will be developed, but the final solution may have different approach, I'll remove ports from deplyoment.yaml file for agreement-consumer and catalog-consumer.